frontend/src/features/projects/spl/steps/SplStep.tsx

```diff
@@ -153,14 +153,14 @@ const SplStep = ({ formikRef }: IStepProps) => {
       <Formik
         initialValues={initialValues}
         validateOnMount={true}
-        enableReinitialize={true}
-        onSubmit={(values: IProject) => {
+        onSubmit={(values: IProject, actions) => {
           return onSubmitReview(
             values,
             formikRef,
             submitStatusCode,
             getStatusTransitionWorkflow(submitStatusCode),
           ).then((project: IProject) => {
+            actions.setValues(project);
```

Comment:
This is required to sync the db project with the form (which will update the rowVersion at the lease).

Reply:
We need to remember this in all our forms, as we need to implement optimistic concurrency.